helper/TabManager.php

```
@@ -0,0 +1,62 @@
+    public function renderPanels($resource, $contentRegion = 'main', $layout = 'vertical')
+    {
+        $view = $this->getView();
+        $tabContentBlocksArray = $view->resourcePageBlocks($resource, $contentRegion)->getBlocksArray();
+        return $view->partial('common/tab-panels.phtml', [
+            'resource' => $resource,
+            'resourcePageBlockArray' => $tabContentBlocksArray,
+            'tabLayout' => $layout
+        ]);
+    }
+
+    public function renderTabsOnly($resource, $contentRegion = 'main', $layout = 'vertical') 
```

PHPMD (2.15.0)

Avoid unused parameters such as '$layout'. (Unused Code Rules)

(UnusedFormalParameter)